var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -80,6 +80,7 @@ class Hdf5(CMakePackage):
             multi=False)
 
     depends_on('cmake@3.12:', type='build')
+    depends_on('gmake', type='build')     # the CMakeFile calls `make`
```

Comment:
Is the makefile named GNUmakefile? If not, it likely works with all `make` executables, it doesn't have to specifically be `gmake`. Spack has a dependency on `make` so we don't add a dep on `gmake` just because a package uses makefiles.

Reply:
Ok.